### ISSUE TITLE
sway window manager

### ISSUE THREAD

**Issue by nobodyatandnothing** [clearlinux/distribution#2545]
- [x] **main:**                      [sway](https://github.com/clearlinux/clr-bundles/blob/master/bundles/sway) -- please undeprecate
- [ ] **terminal:**                   [foot](https://codeberg.org/dnkl/foot) 
https://github.com/clearlinux/distribution/issues/2577
- [ ] **launcher:**                  [fuzzel](https://codeberg.org/dnkl/fuzzel)
          - https://github.com/clearlinux/distribution/issues/1396
          - wofi is marked as "not actively maintained" by it's developer.

- [ ] **clipboard manager:** [wl-clipboard](https://github.com/bugaevc/wl-clipboard)
- [ ] **greeter:**                   [greetd](https://git.sr.ht/~kennylevinsen/greetd)
- [ ] **notification daemon:** [fnott](https://codeberg.org/dnkl/fnott)
- [x] **xdg-utils:**  https://github.com/clearlinux/distribution/issues/2555
- [x] **media player:** [mpv](https://github.com/clearlinux/clr-bundles/blob/master/bundles/mpv)
https://github.com/clearlinux/distribution/issues/2607
- [x] **file manager:** [vifm](https://github.com/clearlinux/clr-bundles/blob/master/bundles/vifm)
https://github.com/clearlinux/distribution/issues/2607
- [ ] **libinput tools:**  https://wayland.freedesktop.org/libinput/doc/latest/tools.html

**Comment by nobodyatandnothing** [comment]
https://github.com/clearlinux/distribution/issues/2571

**Comment by ldelossa** [comment]
Hey @nobodyatandnothing has the "xdg-desktop-portal-wlr" project been added as well? As of today, this is the only way to screen share in applications like Zoom using the web browser.

**Comment by nobodyatandnothing** [comment]
i am not using clear linux at the moment.
the best way to check if software is packaged in clear linux is to search the package name on https://clearlinux.org/software, or do `swupd search <package name>`
you can use [autospec](https://docs.01.org/clearlinux/latest/guides/clear/autospec.html#autospec) to package software for swupd.

**Comment by ldelossa** [comment]
Yup, i know its not there, my message was more of a prod that it should be present in the default sway bundle I believe. I can make a separate issue for this :)
